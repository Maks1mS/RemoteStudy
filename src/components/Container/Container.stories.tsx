import Container from './Container'

export default {
  component: Container,
  title: 'Container'
}

export const text = (): JSX.Element => <Container></Container>
