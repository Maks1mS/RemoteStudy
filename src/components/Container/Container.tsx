import styled from '@emotion/styled'

const Block = styled.div`
  background: white;
  min-height: 50px;
  border-radius: 5px;
  padding: 5px;
`

const Container: React.FC = ({ children, ...props }) => <Block {...props}>{children}</Block>

export default Container
