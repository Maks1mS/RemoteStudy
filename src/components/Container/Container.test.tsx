import { render } from '@testing-library/react'
import Container from './Container'

test('Renders', () => {
  const { container } = render(<Container>Test</Container>)
  expect(container).toMatchSnapshot()
})
