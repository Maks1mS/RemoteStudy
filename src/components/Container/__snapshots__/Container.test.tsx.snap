// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div>
  <div
    class="css-nrfu13-Block e7drqi50"
  >
    Test
  </div>
</div>
`;
